### PR TITLE
add manyToOne

### PR DESCRIPTION
**Pull request by danielnaumau**

### CODE REVIEW

**Comment by Donatas-L** [comment]
Please update README as well @danielnaumau

File: src/main/resources/proto/v2/TimeFilterFastResponse.proto [hunk 1]
```diff
@@ -13,70 +13,81 @@ message TimeFilterFastResponse {
...
+     */
+    DISTANCE_PROPERTY_NOT_SUPPORTED = 12;
+
+    /*
+     * ManyToOne And OneToMany cannot be sent at the same time
+     */
+    BOTH_MANY_TO_ONE_AND_ONE_TO_MANY_CANNOT_BE_SENT = 13;
+
+    /*
+     * ManyToOne And OneToMany cannot be sent at the same time
```
Comment (PKJonas) [inline src/main/resources/proto/v2/TimeFilterFastResponse.proto:88]: Comment doesn't match the error. Could say "ManyToOne and OneToMany cannot both be null at the same time"

Comment (danielnaumau) [reply src/main/resources/proto/v2/TimeFilterFastResponse.proto:88]: Oops, I didn't notice

**Review by PKJonas — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -249,25 +249,23 @@ The request parameters are much more limited and only travel time is returned. I
...
-* origin: Origin point.
-* destination: Destination points. Cannot be more than 200,000.
-* transportation: Transportation type.
+* origin: Origin point;
+* destination: Destination points. Cannot be more than 200,000;
+* transportation: Transportation type;
 * travelTime: Time limit;
-* country: Return the results that are within the specified country
+* country: Return the results that are within the specified country;
+* requestType: MANY_TO_ONE(single departure location and multiple arrival locations) or ONE_TO_MANY (single arrival location and multiple departure locations).
```
Comment (Donatas-L) [inline README.md:257]: This is mixed up.

one-to-many -> single departure location and multiple arrival locations
many-to-one -> single arrival location and multiple departure locations

**Pull request merged**